lib/simulator-xcode-9.js

```diff
@@ -6,6 +6,7 @@ import AsyncLock from 'async-lock';
 import log from './logger';
 import { shutdown as simctlShutdown, bootDevice, eraseDevice } from 'node-simctl';
 import { waitForCondition } from 'asyncbox';
+import { validCoordinates } from './utils';
```

Comment:
it is not necessary to create a separate utility method. Just merge with the recent master and put one more verification into _verifyDevicePreferences_ method